app/src/main/java/com/zeapo/pwdstore/LaunchActivity.kt

```diff
@@ -8,16 +8,13 @@ import android.content.Intent
 import android.os.Bundle
 import android.os.Handler
 import androidx.appcompat.app.AppCompatActivity
-import androidx.core.content.edit
-import androidx.preference.PreferenceManager
 import com.zeapo.pwdstore.crypto.DecryptActivity
-import com.zeapo.pwdstore.utils.BiometricAuthenticator
-import com.zeapo.pwdstore.utils.PreferenceKeys
 
 class LaunchActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+/*
```

Comment:
`LaunchActivity`'s only purpose is auth, if it can't do that then get rid of it entirely and migrate the remaining logic to `AuthActivity`.

Reply:
I'm not so sure about this tbh. The AuthActivity is launched as a new Task and it launches on top of any existing activities. I think it would be better if we keep AuthActivity seperate.